nltk/tokenize/stanford_segmenter.py

```diff
@@ -70,9 +70,8 @@ def __init__(self,
                 searchpath=(), url=_stanford_url,
```

Comment:
I could modify these! And I think we could write a version checker or something to help.